src/utils/IdealGraphVisualizer/View/src/main/java/com/sun/hotspot/igv/view/EditorTopComponent.java

```diff
@@ -101,32 +104,22 @@
         @Override
         public void export(File f) {
 
-            Graphics2D svgGenerator = BatikSVG.createGraphicsObject();
-
-            if (svgGenerator == null) {
-                NotifyDescriptor message = new NotifyDescriptor.Message("For export to SVG files the Batik SVG Toolkit must be intalled.", NotifyDescriptor.ERROR_MESSAGE);
-                DialogDisplayer.getDefault().notifyLater(message);
-            } else {
-                scene.paint(svgGenerator);
-                FileOutputStream os = null;
-                try {
-                    os = new FileOutputStream(f);
-                    Writer out = new OutputStreamWriter(os, "UTF-8");
-                    BatikSVG.printToStream(svgGenerator, out, true);
-                } catch (FileNotFoundException e) {
-                    NotifyDescriptor message = new NotifyDescriptor.Message("For export to SVG files the Batik SVG Toolkit must be intalled.", NotifyDescriptor.ERROR_MESSAGE);
-                    DialogDisplayer.getDefault().notifyLater(message);
-
-                } catch (UnsupportedEncodingException e) {
-                } finally {
-                    if (os != null) {
-                        try {
-                            os.close();
-                        } catch (IOException e) {
-                        }
+            SVGGraphics2D svgGenerator = createGraphicsObject();
+            scene.paint(svgGenerator);
+            FileOutputStream os = null;
+            try {
+                os = new FileOutputStream(f);
+                Writer out = new OutputStreamWriter(os, "UTF-8");
+                printToStream(svgGenerator, out, true);
+            } catch (IOException e) {
+                e.printStackTrace();
+            } finally {
+                if (os != null) {
+                    try {
+                        os.close();
+                    } catch (IOException e) {
                     }
                 }
-
             }
```

Comment:
Can be simplified to
```
try (FileOutputStream os = new FileOutputStream(f)) {
    Writer out = new OutputStreamWriter(os, StandardCharsets.UTF_8);
    printToStream(svgGenerator, out, true);
} catch (IOException e) {
    e.printStackTrace();
}
```
You could also use `StandardCharsets.UTF_8` instead of `"UTF-8"`.